components/toast_wrapper/toast_wrapper.jsx

```diff
@@ -73,6 +71,10 @@ class ToastWrapper extends React.PureComponent {
             showUnreadToast = unreadCount > 0 && !props.atBottom;
         }
 
+        if (typeof showMessageHistoryToast === 'undefined' && props.focusedPostId !== '') {
```

Comment:
if focusedPostId is empty, show messageHistoryToast can be `undefined`, is this expected?

Reply:
Yes, it can be that means it was not set Previously. Ideally i would have preferred to have this in mount by not mounting this component till it is needed. But i cannot do that because of this particular issue and the shortcomings of virt list.